radvd/start.sh

```diff
@@ -1,15 +1,24 @@
 #!/bin/bash
 
-ROUTER_INTERFACE='eth0' # Device's physical interface on the local network (eth0=ethernet, wlan0=WiFi typically)
-ROUTED_PREFIX='2001:1122:ABCD:BABA::/64' # Tunnel's "routed prefix", to be advertised to the local network
-TUNNEL_PREFIX='2001:2233:EEFF:1234::/64' # Tunnel's own subnet, from which ::1 (gateway) and ::2 (client) are derived
+# Device's physical interface on the local network (eth0=ethernet, wlan0=WiFi typically)
+ROUTER_INTERFACE="${ROUTER_INTERFACE:-eth0}"
+# Tunnel's "routed prefix", to be advertised to the local network
+ROUTED_PREFIX="${ROUTED_PREFIX:-001:db8:1::/64}"
+TUNNEL_PREFIX="${TUNNEL_PREFIX:-001:db8:2::/64}"
```

Comment:
Here as well, and elsewhere too, `001` -> `2001`.

The best though, imo, would be to have a default of empty value (with a realistic-looking dummy sample value _as a comment_ to help the reader), and have the script print an error message in a sleep loop (to avoid busy restarting of the container by the supervisor):
```
while true; do
   if [ -z "$TUNNEL_PREFIX" ]; then
        echo Error: environment variable TUNNEL_PREFIX is empty.
        echo Please set the env vars through the web dashboard. See README.
    else
        break
    fi
    sleep 10
done
```
But something cleverer than the above to avoid copy and paste for each single env variable (maybe listing all variable names in an array and then using [indirect expansion](https://stackoverflow.com/questions/8515411/what-is-indirect-expansion-what-does-var-mean)).

I understand that, when the user sets the value through the dashboard, the supervisor restarts the container, thus breaking this loop (and the whole process).

Reply:
I didn't test, my understanding (from reading support threads, forums, FD over time...) was that the supervisor is notified straight away through the VPN "when things change" (what things exactly, not sure), and in addition to that, the supervisor would regularly also poll the API.